src/win/tcp.c

```diff
@@ -1280,6 +1280,10 @@ int uv_tcp_nodelay(uv_tcp_t* handle, int enable) {
 }
 
 
+int uv_tcp_fastopen(uv_tcp_t* handle, int enable) {
+  return UV_ENOTSUP;
+}
+
```

Comment:
Apparently it's supported [starting with Windows 10, version 1607](https://docs.microsoft.com/en-us/windows/win32/winsock/ipproto-tcp-socket-options)